.devcontainer/compose.yaml

```diff
@@ -55,6 +55,24 @@ services:
     networks:
       - internal_network
 
+  valkey:
+    image: valkey/valkey:8-alpine
+    restart: unless-stopped
+    volumes:
+      - valkey-data:/data
+    networks:
+      - internal_network
+
+  dragonfly:
+    image: docker.dragonflydb.io/dragonflydb/dragonfly
+    restart: unless-stopped
+    volumes:
+      - dragonfly-data:/data
+    networks:
+      - internal_network
+    ulimits:
+      memlock: -1
+
```

Comment:
I must admit I do not use devcontainers and am only vaguely familiar with how they work. But does this mean that containers for valkey and dragonfly are always spun up? If yes, that would seem wasteful. Is it really necessary?